workspace/notebook/py_sb_horizon_harmonised_nsip_project.json

```diff
@@ -312,7 +312,7 @@
 					"                        ,applicantwebaddress\n",
 					"                        ,applicantphonenumber\n",
 					"                        ,applicantdescriptionofproject\n",
-					"                        ,caseNodeId\n",
+					"                       --- ,caseNodeId\n",
```

Comment:
We are losing a column, HorizonCaseNumber, I think it needs to be added to the query

Reply:
updated and included HorizonCaseNumber